mp/src/game/client/momentum/ui/HUD/hud_speedometer.cpp

```diff
@@ -26,93 +31,160 @@ using namespace vgui;
 static MAKE_TOGGLE_CONVAR(mom_hud_speedometer, "1", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
                           "Toggles displaying the speedometer. 0 = OFF, 1 = ON\n");
 
-static MAKE_TOGGLE_CONVAR(
-    mom_hud_speedometer_hvel, "0", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
-    "Toggles showing only the horizontal component of player speed. 0 = OFF (XYZ), 1 = ON (XY)\n");
-
 static MAKE_CONVAR(mom_hud_speedometer_units, "1", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
                    "Changes the units of measurement of the speedometer.\n 1 = Units per second\n 2 = "
                    "Kilometers per hour\n 3 = Miles per hour\n 4 = Energy",
                    1, 4);
 
 static MAKE_CONVAR(mom_hud_speedometer_colorize, "1", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
-                          "Toggles speedometer colorization. 0 = OFF, 1 = ON (Based on acceleration)," 
-                          " 2 = ON (Staged by relative velocity to max.)\n", 0, 2);
+                   "Toggles speedometer colorization. 0 = OFF, 1 = ON (Based on acceleration),"
+                   " 2 = ON (Staged by relative velocity to max.)\n", 0, 2);
 
-static MAKE_TOGGLE_CONVAR(mom_hud_speedometer_showlastjumpvel, "1", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
+static MAKE_TOGGLE_CONVAR(mom_hud_speedometer_unit_labels, "0", FLAG_HUD_CVAR,
+                          "Toggles showing the unit value labels (KM/H, MPH, Energy, UPS). 0 = OFF, 1 = ON\n");
+
+static MAKE_TOGGLE_CONVAR(mom_hud_speedometer_showenterspeed, "1", FLAG_HUD_CVAR,
+    "Toggles showing the stage/checkpoint enter speed (and comparison, if existent). 0 = OFF, 1 = ON\n");
+
+static MAKE_TOGGLE_CONVAR(mom_hud_speedometer_horiz, "1", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
+                          "Toggles displaying the speedometer. 0 = OFF, 1 = ON\n");
+
+static MAKE_TOGGLE_CONVAR(mom_hud_speedometer_lastjumpvel, "1", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
                           "Toggles showing player velocity at last jump (XY only). 0 = OFF, 1 = ON\n");
 
-static MAKE_CONVAR(mom_hud_speedometer_lastjumpvel_fadeout, "3.0", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE, 
+static MAKE_CONVAR(mom_hud_speedometer_lastjumpvel_fadeout, "3.0", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
                    "Sets the fade out time for the last jump velocity.", 1.0f, 10.0f);
 
-static MAKE_TOGGLE_CONVAR(
-    mom_hud_speedometer_showenterspeed, "1", FLAG_HUD_CVAR,
-    "Toggles showing the stage/checkpoint enter speed (and comparison, if existent). 0 = OFF, 1 = ON\n");
+static MAKE_CONVAR(mom_hud_velocity_type, "0", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
+                   "Toggles the velocity type used in comparisons and map finished dialog. 0 = ABSOLUTE, 1 = HORIZONTAL",
+                   0, 1);
```

Comment:
```suggestion
static MAKE_TOGGLE_CONVAR(mom_hud_velocity_type, "0", FLAG_HUD_CVAR | FCVAR_CLIENTCMD_CAN_EXECUTE,
                   "Toggles the velocity type used in comparisons and map finished dialog. 0 = ABSOLUTE, 1 = HORIZONTAL");
```

Reply:
Reasoning for this was to make room for the possibility of different speed types like vertical only, but that is unlikely to happen anyway. Made the requested change.